packages/bsky/src/db/migrations/20231220T225126090Z-blob-takedowns.ts

```diff
@@ -0,0 +1,61 @@
+import { Kysely } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .createTable('blob_takedown')
+    .addColumn('did', 'varchar', (col) => col.notNull())
+    .addColumn('cid', 'varchar', (col) => col.notNull())
+    .addColumn('takedownId', 'varchar', (col) => col.notNull())
+    .addPrimaryKeyConstraint('blob_takedown_pkey', ['did', 'cid'])
+    .execute()
+  await db.schema
+    .alterTable('actor')
+    .dropConstraint('actor_takedown_id_fkey')
+    .execute()
+  await db.schema
+    .alterTable('record')
+    .dropConstraint('record_takedown_id_fkey')
+    .execute()
+  await db.schema
+    .alterTable('actor')
+    .alterColumn('takedownId')
+    .setDataType('varchar')
```

Comment:
may require a bit of care doing this in prod - but i'm pretty sure we can do the int -> string cast pretty straightforwardly

Reply:
Do we prefer it over adding a new column `takedownRef`?  That was pretty smooth last time and has the side benefit of being consistent with the pds.

Reply:
ah right i forgot about that approach... i'm game 👍